bin/get-all-tasks

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+## This script get all tasks for SAPS interacting with Dispatcher backend.
+
+## checking args
+if [ $# -ne 3 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=$1
+readonly USER_PASSWORD=$2
+readonly DISPATCHER_ADDRESS=$3
+
+ALL_TASKS=$(curl --silent --request GET "${DISPATCHER_ADDRESS}/processings" --header "Content-Type: text/plain"  --header "userEmail: $USER_EMAIL"  --header "userPass: $USER_PASSWORD")
+JSON_LENGTH=$(($(echo $ALL_TASKS | jq 'length') - 1))
+
+declare -a FILTERS_ARRAY=("taskId" "dataset" "region" "inputGatheringTag" "inputPreprocessingTag" "algorithmExecutionTag" "state" "creationTime" "updateTime")
+FILTERS_ARRAY_LENGTH=$((${#FILTERS_ARRAY[@]} - 1))
+
+echo "${FILTERS_ARRAY[@]}"
+
+for json_index in $(seq 0 $JSON_LENGTH); do
```

Comment:
this loop is complex. add a comment explaining what it does

Reply:
Done